testing/testPresetData.js

```diff
@@ -55,6 +55,7 @@ const cookInfoResult = {
   cook_summary: 'Delicious recipies passed down 3 generations of Afifi',
   cook_image_url: 'https://user-images.githubusercontent.com/28222381/45268630-08251500-b488-11e8-94c4-7e664f5d45c9.jpg',
   count_reviews: '3',
+  av_reviews: '4.0000000000000000',
```

Comment:
same

Reply:
Where is this being tested?

Reply:
It's been used as part of testing the query.